packages/react/src/drafts/IssueLabel/getColorFromHex.ts

```diff
@@ -0,0 +1,137 @@
+import {getContrast} from 'color2k'
+import {Hsluv} from 'hsluv'
+
+/**
+ * transforms a hex color provided by the user into a color object with background and text colors
+ * @param colorHex — the hex color provided by the user
+ * @param colorScheme — the color scheme the user has currently set
+ * @param isSelected
+ * @param bgColor — the background color from the selected theme, needed to calc the contrast for the colors
+ * @returns
+ */
+export const getColorsFromHex = (
+  colorHex: `#{string}`,
+  colorScheme: string = 'light',
+  bgColor: string,
+): React.CSSProperties | undefined => {
```

Comment:
Some ideas for the types on this to make them more specific:

```suggestion

type Hex = `#${string}`;

/**
 * transforms a hex color provided by the user into a color object with background and text colors
 * @param colorHex — the hex color provided by the user
 * @param colorScheme — the color scheme the user has currently set
 * @param isSelected
 * @param bgColor — the background color from the selected theme, needed to calc the contrast for the colors
 * @returns
 */
export const getColorsFromHex = (
  colorHex: Hex,
  colorScheme: 'light' | 'dark' = 'light',
  bgColor: Hex,
): React.CSSProperties | undefined => {
```

Reply:
Hey @joshblack I removed `'light' | 'dark'` because it could be any of the color schemes, also `dark_colorblind`, etc.